### PR TITLE
Added a description attribute to Field class #124

### PR DESCRIPTION
**Pull request by FirstKlaas**
Added a description attribute to Field class as proposed in issue #124  

## Description
Added an attribute (with type hint Optional[str]) to the class Field. It Defaults to None

## Motivation and Context
It's the first time for me contributing to a project like this. So I took an "good first issue" to learn the whole process. The Issue #124 seemed to be a good starting point for me. 

## How Has This Been Tested?
I called make check and make test.

## Checklist:
- [X ] My code follows the code style of this project.
- [ X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Hope I did it right and I can tackle the next Issue. I there is anything I can improve, let me know.

### CODE REVIEW

**Comment by grigi** [comment]
Yay!
It does just about cover it :-)

Please add an entry in CHANGELOG for 0.12.3 with a quick description of the change, and add yourself to the CONTRIBUTORS

**Comment by grigi** [comment]
Also, please add the same short description in the docs: https://github.com/tortoise/tortoise-orm/blob/master/docs/fields.rst#reference

**Comment by FirstKlaas** [comment]
Cool. Should I add the entry in the CHANGELOG in my fork/branch (and commit
and push again)? And do I have to do another pull request? Sorry about the
noob question.

Cheers!

Am Di., 25. Juni 2019 um 17:31 Uhr schrieb Nickolas Grigoriadis <
notifications@github.com>:

**Comment by grigi** [comment]
Add it in this branch. That way the work and docs are managed as one thing.
And don't mind asking questions :-)

**Comment by FirstKlaas** [comment]
What do you think, should I add a Testcase for the description attribute? I would extends the Testmodels to have a Model with two fields one with and one without description. Also adding a test case in the tortoise.test.fields package.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Changelog
 0.12.2
 ------
 * Fix accidental double order-by for ``.values()`` based queries. (#143)
+* Added description attribute to Field class. (#124)
```
Comment (grigi) [inline CHANGELOG.rst]: Create a 0.12.3 section please. 0.12.2 is already released.

**Comment by grigi** [comment]
I really don't see the value in doing a test that a field that was added, was, in fact, added.
Other than it would test that it wasn't removed...
But that leads to a rabbit hole, we don't have any such "interface stability" tests at this stage. If you feel such a thing is useful you can do it, it is optional.

File: CHANGELOG.rst (hunk 1 above)
Comment (FirstKlaas) [reply CHANGELOG.rst]: You are right. Changed it.

**Comment by FirstKlaas** [comment]
@grigi I agree. Wanted to comply your standards.

**Comment by grigi** [comment]
Thanks for your work Klaas :+1:

**Pull request merged**